src/current/v26.1/physical-cluster-replication-technical-overview.md

```diff
@@ -68,3 +68,9 @@ When a PCR stream is started with a `readonly` virtual cluster, the job will del
 After reverting any necessary data, the standby virtual cluster is promoted as available to serve traffic and the replication job ends.
 
 For details on failing back to the primary cluster following a failover, refer to [Fail back to the primary cluster]({% link {{ page.version.version }}/failover-replication.md %}#failback).
+
+## Multi-region behavior
```

Comment:
Is this supposed to be an `<h3>` instead? Noticed in the preview that `Multi-region behavior` isn't nested with everything else:

<img width="271" height="235" alt="Image" src="https://github.com/user-attachments/assets/c240bed6-8650-4d49-85cf-032ceb5ba599" />